### ISSUE TITLE
Enumerated list rendering broken

### ISSUE THREAD

**Issue by dreeves** [beeminder/blog#387]
Spotted in https://blog.beeminder.com/akrasia

![image](https://github.com/beeminder/blog/assets/9928/d65f61c7-a78d-48c0-9312-e8f241c8143a)

## Cognata

* #385 
* #389

_**Verbata**: mendoza, enumerated lists, numbered lists, markdown rendering,_

**Comment by dreeves** [comment]
I haven't investigated this yet but intend to apply a workaround ASAP since this is pretty embarrassing brokenness that makes the post quasi-unreadable.

**Comment by narthur** [comment]
@dreeves Yes, I think this is my fault. I deployed a fix to the  mid-paragraph comments issue and I think that broke a lot of things the way I did it.

Today I've added snapshot testing to help me review how changes impact the backlog to hopefully mitigate the risk of this kind of thing in the future.

In the meantime I'll go ahead and roll back the comment fix since I think my fix created way more problems than it solved.
